src/EditorFeatures/TestUtilities2/Intellisense/TestStateBase.vb

```diff
@@ -24,6 +25,30 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
         Protected ReadOnly SignatureHelpAfterCompletionCommandHandler As SignatureHelpAfterCompletionCommandHandler
         Private ReadOnly FormatCommandHandler As FormatCommandHandler
 
+        Private Shared s_lazyEntireAssemblyCatalogWithCSharpAndVisualBasicWithoutCompletionTestParts As Lazy(Of ComposableCatalog) =
+            New Lazy(Of ComposableCatalog)(Function()
+                                               Return TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.
+                                               WithoutPartsOfTypes({
+                                                                   GetType(IIntelliSensePresenter(Of ICompletionPresenterSession, ICompletionSession)),
+                                                                   GetType(IIntelliSensePresenter(Of ISignatureHelpPresenterSession, ISignatureHelpSession)),
+                                                                   GetType(FormatCommandHandler)}).
+                                               WithParts({
+                                                         GetType(TestCompletionPresenter),
+                                                         GetType(TestSignatureHelpPresenter),
+                                                         GetType(IntelliSenseTestState)})
+                                           End Function)
+
+        Private Shared EntireAssemblyCatalogWithCSharpAndVisualBasicWithoutCompletionTestParts As ComposableCatalog =
```

Comment:
💭 If this isn't a property, why bother with the `Lazy<T>`?

Reply:
Sure. Need to wrap these as properties. No need to initialize them for non-default calls.

---
In reply to: [262669770](https://github.com/dotnet/roslyn/pull/33817#discussion_r262669770) [](ancestors = 262669770)

Reply:
They are static, so they are only initialized once anyway. Unlikely any value would ever be derived from making them lazy.

Reply:
Correction: The one bit of value is failures, if any, would not occur during the static constructor. This may be enough to justify lazy construction.